diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -118,24 +118,30 @@ namespace diff_drive_controller
      * \brief Wheel speed limiter setter
      * \param wheel_speed_limiter Wheel speed limiter
      */
-    void setWheelSpeedLimiter(WheelSpeedLimiter wheel_speed_limiter)
```

Comment:
The other one can't be inlined anyway, since it's virtual (there will always be a function pointer).

This one I don't think matters, since it's only called once (or never).